dev/helm/values.yaml

```diff
@@ -80,9 +80,25 @@ postgresql:
   ## Set to false if bringing your own PostgreSQL, and set secret value postgresql-uri.
   ##
   enabled: true
+  ## ssl enforce SSL communication with PostgresSQL
+  ## Default to false
+  ##  
+  # ssl: false
+  ## ca Certificate of Authority
+  ## Default to empty, point to location of CA
+  ##  
+  # ca: "path to ca"
```

Comment:
Should be added to the Configuration section of the README as well.

Reply:
Added notes in the REAME on using the ssl/ca settings.